content/fr/docs/what-is-opentelemetry.md

```diff
@@ -8,10 +8,9 @@ default_lang_commit: 71833a5f8b84110dadf1e98604b87a900724ac33
 
 OpenTelemetry, c'est :
 
-- Un framework
-  d'[observabilité](/docs/concepts/observability-primer/#what-is-observability)
-  et des outils conçus pour créer et gérer des données de télémétrie telles que
-  des [traces](/docs/concepts/signals/traces/), des
+- Un framework d'[observabilité](/docs/concepts/observability-primer/) et des
+  outils conçus pour créer et gérer des données de télémétrie telles que des
+  [traces](/docs/concepts/signals/traces/), des
```

Comment:
This, and similar changes in this page that remove `#what-is-observability` from links, need to be undone.